plugins/fs/Cargo.toml

```diff
@@ -13,6 +13,13 @@ links = "tauri-plugin-fs"
 rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "No write access to `$RESOURCES` folder" }
+android = { level = "partial", notes = "Access is restricted to Application folder by default" }
+ios = { level = "partial", notes = "Access is restricted to Application folder by default" }
```

Comment:
@lucasfernog is this correct and the "only" limitation? This is partial as it does not implement all desktop APIs